src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.Scanner;
+
+public class Storage {
+    private String filepath;
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    public ArrayList<Task> load() throws FileNotFoundException {
+        File f = new File(this.filepath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        ArrayList<Task> userInputs = new ArrayList<>();
+        while (s.hasNextLine()) {
+            String string = s.nextLine();
+            String[] items = string.split(" , ");
+            String task = items[0];
+            boolean flag = Objects.equals(items[1], "1");
+            if (Objects.equals(task, "T")) {
```

Comment:
I was thinking it might be a bit more readable to use the `string.equals()` method to compare strings.